nodes/src/nodes/preprocessor_native/tests/test_indexer.cpp

```
@@ -0,0 +1,195 @@
+
+void test_thread_safety() {
+    rr::Indexer idx;
+    constexpr int num_writers = 4;
+    constexpr int docs_per_writer = 100;
+    constexpr int num_readers = 4;
+    constexpr int searches_per_reader = 100;
+
+    std::vector<std::thread> threads;
+
+    // Writer threads
+    for (int w = 0; w < num_writers; ++w) {
```

Clang (14.0.6)

[warning] loop variable name 'w' is too short, expected at least 2 characters

(readability-identifier-length)